modules/client-configuration/pages/snippets/trust_gpg.adoc

```diff
@@ -32,5 +75,5 @@ You do not need to delete any previously stored keys.
 ====
 Trusting a GPG key is important for security on clients. 
 It is the task of the admin to decide which keys are needed and can be trusted.
-Trusting the key is done manually,  either by writing a Salt state or adding the keys to the bootstrap script.
+Because a software channel cannot be used when the GPG key is not trusted, the decision of assigning a channel to a client depends on the decision of trusting the key.
```

Comment:
A software channel cannot be used when a GPG key is not trusted. Assigning a channel to a client depends on if its GPG key has been trusted.